README.md

```diff
@@ -15,21 +17,22 @@ Please visit the [official Parasoft website](http://www.parasoft.com) for more i
 ## Quick start
 
 To analyze your code with Parasoft dotTEST and review analysis results on GitHub, you need to customize your GitHub workflow to include:
- - The action to run dotTEST analyzer.
- - The action to upload the dotTEST analysis report in the SARIF format to GitHub. The SARIF report is generated by default when the dotTEST action is run.
+ - The action to run dotTEST.
+ - The action to upload the dotTEST analysis report in the SARIF format to GitHub.
+ - The action to upload the dotTEST analysis reports in other formats (XML, HTML, etc.) to GitHub as workflow artifacts.
 
 ### Prerequisites
 This action requires Parasoft dotTEST with a valid Parasoft license.
 
 We recommend that you run Parasoft dotTEST on a self-hosted rather than GitHub-hosted runner.
 
 ### Adding the dotTEST Action to a GitHub Workflow
-Adding the action to run dotTEST analyzer to your workflow allows you to launch code analysis with Parasoft dotTEST.
+Add the `Run dotTEST` action to your workflow to launch code analysis with Parasoft dotTEST.
```

Comment:
`Run dotTEST` action - I think the official name according to e-mail from Mirek is `Run Parasoft dotTEST`

Reply:
You're right. Fixed.